pom.xml

```diff
@@ -226,8 +226,8 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <groupId>pl.project13.maven</groupId>
-                    <artifactId>git-commit-id-plugin</artifactId>
+                    <groupId>io.github.git-commit-id</groupId>
```

Comment:
Is this actually related to this change?

Reply:
Yes, we need this Version if we want to use JDK17, which is needed for the jpackage.